app/assets/javascripts/bastion/components/bst-alert.directive.js

```diff
@@ -10,16 +13,25 @@
  *     <div bst-alert="success"></div>
  *   </pre>
  */
-angular.module('Bastion.components').directive('bstAlert', function () {
+angular.module('Bastion.components').directive('bstAlert', ['$animate', '$timeout', function ($animate, $timeout) {
+    var SUCCESS_FADEOUT = 5000;
```

Comment:
If I'm looking in the [correct place](https://github.com/theforeman/foreman/blob/develop/vendor/assets/javascripts/jquery.jnotify.js#L38-38) it looks like foreman uses 1 second.  I think this may be too fast to read some of our longer messages though.  What about three seconds?

Ideally we'd not fade out while the message had focus (i.e. mouse was over it).  I can try to implement that and see how difficult it is.